docs/src/examples/components/Menu/Usages/index.tsx

```diff
@@ -24,6 +25,16 @@ const Usages = () => (
       description="When Submenu in MenuItem is controlled, then its 'menuOpen' prop value could be changed either by parent component, or by user actions (e.g. key press) - thus it is necessary to handle 'onMenuOpenChange' event."
       examplePath="components/Menu/Usages/MenuExampleWithSubmenuControlled"
     />
+    <ComponentExample
+      title="Menu with items containing tooltip"
```

Comment:
nit: for the sake of consistency, lets choose either shorter `With tooltip` form (which is used for `Button` - i.e. format where component's name is omitted), or use the long form with component name. Personally, think that `With tooltips` should be enough to describe the example